label_sync/labels.yaml

```diff
@@ -1814,6 +1814,144 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to CAPBK
+        name: area/kubeadm-bootstrap
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to KCP
+        name: area/kubeadm-control-plane
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to infrastructure docker
+        name: area/infrastructure-docker
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0052cc
```

Comment:
we seem to have two e2e area's

Reply:
In the current list there's no distinction - probably a good one to add to the list though!

Reply:
Acknowledged, and updated.